test.js

```diff
@@ -2,6 +2,8 @@ const lingo = require("./index");
 const assert = require("assert");
 const config = require("./test_config");
 
+lingo.baseURL = "https://api-test.lingoapp.com/alpha";
```

Comment:
Maybe this value should go in `test_config`? Any outside contributor will not be able to reach this URL, which would probably be confusing (and even for us, it might be useful to have different test configs that map to different environments). In the future we may want to include eg [`nock`](https://github.com/nock/nock) to return mock responses.

Reply:
Thought about that too. My main concern now was not forgetting to swap this out in `index.js` after running tests, which I did 🤦‍♂.